components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-overview/overview-status/overview-status.component.html

```diff
@@ -1,6 +1,6 @@
 <chef-radial-chart class="chart">
   <span slot="innerText">
-    {{ data.failed + data.passed + data.skipped | number }} Total {{ type | titlecase }}
+    {{ data.failed + data.passed + data.skipped + data.waived | number }} Total {{ type | titlecase }}
```

Comment:
just a thought - I feel like its favored here to do operations outside of the template.  Might be preferable to create a `getTotal()` function that is simply
```
public getTotal(): number {
    return this.data.failed + this.data.passed + this.data.skipped + this.data.waived;
}
```

Reply:
I rescind my comment!  My bad!

https://medium.com/showpad-engineering/why-you-should-never-use-function-calls-in-angular-template-expressions-e1a50f9c0496